experimental/packages/opentelemetry-instrumentation-http/package.json

```diff
@@ -9,7 +9,7 @@
     "prepublishOnly": "npm run compile",
     "compile": "tsc --build",
     "clean": "tsc --build --clean",
-    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "test": "cross-env NODE_OPTIONS='--max-old-space-size=8192' nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
```

Comment:
This one is here to fix local (cloned repo) builds, it also fixes the CI test run for `unit-test.yml`, but because of other failing tests I've also added line 67 `NODE_OPTIONS: --max-old-space-size=8192` to `unit-test.yml`...